es/baseapi.py

```diff
@@ -120,19 +177,19 @@ def __init__(self, url, es, **kwargs):
         # this is set to an iterator after a successfull query
         self._results = None
 
-    @property
+    @property  # type: ignore
```

Comment:
what was the complaint here?

Reply:
es/baseapi.py:189: error: Decorated property not supported

Reply:
Oh wow, quite the can of worms